src/eduid_userdb/credentials/fido.py

```diff
@@ -50,13 +52,21 @@ class FidoCredential(Credential):
     Token authentication credential
     """
 
-    def __init__(self, data):
+    def __init__(self, data: Dict[str, Any], called_directly: bool = True):
 
-        Credential.__init__(self, data)
+        Credential.__init__(self, data, called_directly=called_directly)
```

Comment:
Should these not be calls to super instead?

Reply:
:+1: